src/test/java/com/flagsmith/FlagsmithTestHelper.java

```diff
@@ -223,18 +223,27 @@ public static int createProject(String name, int organisationId) {
         .getInt("id");
   }
 
-  public static Flag flag(String name, String description, boolean enabled) {
+  public static Flag flag(String name, String description, String type, boolean enabled, String value) {
     final com.flagsmith.Feature feature = new com.flagsmith.Feature();
     feature.setName(name);
     feature.setDescription(description);
-    feature.setType("FLAG");
+    feature.setType(type);
 
     final Flag result = new Flag();
     result.setFeature(feature);
     result.setEnabled(enabled);
+    result.setStateValue(value);
     return result;
   }
 
+  public static Flag flag(String name, String description, boolean enabled) {
+    return flag(name, description, "FLAG", enabled, null);
+  }
+
+  public static Flag typelessFlag(String name, boolean enabled, String value) {
```

Comment:
We shouldn't have a concept of a `typelessFlag` - all features must have a type in the API so I'm not sure why we would have something different in the SDKs?

Reply:
What are the possible flag types the Flagsmith API can return, and which one should we use as default?

Reply:
I added the logic above.

Reply:
As per the discussion on #47, this is fine for now but we will, in the near future remove the `type` attribute from the `Flag` class.